vnext/Microsoft.ReactNative/ReactPointerEventArgs.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ReactPointerEventArgs.g.h"
+#include <UI.Xaml.Input.h>
+
+namespace winrt::Microsoft::ReactNative::implementation {
+struct ReactPointerEventArgs : ReactPointerEventArgsT<ReactPointerEventArgs> {
+  xaml::Input::PointerRoutedEventArgs Args() const noexcept;
+
+  winrt::IInspectable Target() const noexcept;
+  void Target(winrt::IInspectable const &target) noexcept;
+
+  void PreventDefault();
+  void ReleaseCapture();
+  void StopPropagation();
```

Comment:
Should we remove StopPropagation for now since we don't have a use case for it yet?

Reply:
It also has unclear semantics, since the current implementation will stop bubbling of the event to parent ViewManagers, but does not necessarily prevent the event from being emitted to JS.

Reply:
Yeah I would remove it